packages/radix-ui-themes/src/components/text.css

```diff
@@ -1,8 +1,11 @@
 .rt-Text {
-  margin: 0;
   line-height: var(--line-height, var(--default-line-height));
   letter-spacing: var(--letter-spacing, inherit);
 
+  :where(&) {
+    margin: 0;
+  }
```

Comment:
`.rt-Text` was able to overwrite margin of a component that composes it and has styles before `text.css` is imported

Reply:
Nope, that's only `:where(.rt-Text)`. `&` is where it puts the selector, it won't be appended in the front.

Reply:
Ah yeah of course I misread.